yarn.lock

```diff
@@ -380,14 +380,26 @@
   resolved "https://registry.npmjs.org/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@cspotcode/source-map-consumer@0.8.0":
```

Comment:
Did we decide to no longer include lock files in individual PR branches?

Reply:
We didn't yet. We had the discussion but no clear conclusion. It would be my preference to leave them out - they are a deployment/DevOps/security concern not so much day to day development.  But seems most people on the net haven't realized that yet ;-)